src/test/java/tests/RepositorySelectorPopulationTests.java

```diff
@@ -47,16 +49,17 @@ public void noJsonFiles() {
     public void oneInvalidJsonFile() throws IOException {
         File invalidJson = new File("store/test/dummy-dummy.json");
         assert invalidJson.createNewFile();
-        RepoIO testIO = new RepoIO(true, true);
+        RepoIO testIO = TestController.createTestingRepoIO(Optional.of(new JSONStoreStub()));
```

Comment:
From what I understand, this test suite is used to test RepoIO repository list, which is based on the JSON. It tests populating the RepoIO. I don't think it's a `StoreTest`, but now that I relook at it, I agree that all the tests should use the real `JSONStore`.